metrics/web/test/common/presentation/app_bar/widget/metrics_app_bar_test.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/common/presentation/app_bar/widget/metrics_app_bar.dart';
+import 'package:metrics/common/presentation/metrics_theme/config/dimensions_config.dart';
+import 'package:network_image_mock/network_image_mock.dart';
+
+void main() {
+  group("MetricsAppBar", () {
```

Comment:
What do you think about testing whether the app icon is present? Also usually app icon leads to the main page of the app/home - should we implement it as well?